apps/desktop/package.json

```
@@ -23,7 +23,7 @@
   },
   "devDependencies": {
     "@tauri-apps/cli": "^1.2.0",
-    "eslint-config-custom": "workspace:*",
-    "tsconfig": "workspace:*"
+    "eslint-config-custom": "*",
+    "tsconfig": "*"
```

yarn v1 must have messed with it
```suggestion
    "eslint-config-custom": "workspace:*",
    "tsconfig": "workspace:*"
```